src/transformers/models/wavlm/modeling_wavlm.py

```diff
@@ -44,14 +44,23 @@
 
 _CONFIG_FOR_DOC = "WavLMConfig"
 _PROCESSOR_FOR_DOC = "Wav2Vec2Processor"
+
 _CHECKPOINT_FOR_DOC = "patrickvonplaten/wavlm-libri-clean-100h-base-plus"
+_EXPECTED_OUTPUT_SHAPE = [1, 292, 768]
+_CTC_EXPECTED_OUTPUT = "'mister quilter is the aposle of the middle classes and we are glad to welcome his gospel'"
+_CTC_EXPECTED_LOSS = 12.51
 
-_SEQ_CLASS_CHECKPOINT = "microsoft/wavlm-base"
 _FEAT_EXTRACTOR_FOR_DOC = "Wav2Vec2FeatureExtractor"
 
 _SEQ_CLASS_CHECKPOINT = "microsoft/wavlm-base-plus"
+_SEQ_CLASS_EXPECTED_OUTPUT = "label"  # TODO(anton) - could you quickly fine-tune a KS WavLM Model
+_SEQ_CLASS_EXPECTED_LOSS = 0.69  # TODO(anton) - could you quickly fine-tune a KS WavLM Model
+
 _FRAME_CLASS_CHECKPOINT = "microsoft/wavlm-base-plus-sd"
+_FRAME_EXPECTED_OUTPUT = [0, 0]
 _XVECTOR_CHECKPOINT = "microsoft/wavlm-base-plus-sv"
+_XVECTOR_EXPECTED_OUTPUT = 0.97
```

Comment:
I wonder if this doesn't unnecessarily complexify the file, given that those seem to be used only once?

Reply:
It's mainly so that we are able to keep using the `" # Copied from ` mechanism

Reply:
I see, good for me